code-studio/src/js/components/progress/course_progress_row.jsx

```diff
@@ -66,20 +75,26 @@ const CourseProgressRow = React.createClass({
     const stage = this.props.stage;
 
     let rowStyle = Object.assign({}, styles.row);
-    let ribbon;
+    let ribbon, editFocusArea;
```

Comment:
How do we feel about the word edit? It seems more like change. You aren't altering some trait of your focus area you are picking a different area to focus on.